recipes/_filebeat-git.rb

```diff
@@ -0,0 +1,115 @@
+file "#{node['filebeat']['base_dir']}/filebeat.xml" do
+  action :delete
+end
+
+git_owner = ""
```

Comment:
Change these lines to 
  git_owner = "glassfish"
  git_group = "glassfish"
  
  Then you can remove the else clause in the statement below.